### ISSUE TITLE
Keypress event does not detect "Esc" key

### ISSUE THREAD

**Issue by yuhui** [adobe/reactor-extension-core#31]
### Expected Behaviour
When pressing the Esc key, the Keypress event should detect it.

### Actual Behaviour
The Keypress event does not detect pressing the Esc key.

### Reproduce Scenario (including but not limited to)
N/A

#### Steps to Reproduce
1. Add a rule that includes the Keypress event.
2. In the test web page, press the Esc key.

Validate:
- Press the "Enter" key: Keypress event detects this key.

#### Platform and Version
macOS Big Sur, Google Chrome 87

#### Sample Code that illustrates the problem
N/A

#### Logs taken while reproducing problem
N/A

**Comment by thebenrob** [comment]
Yeah, keypress only handles printable characters, so things like Escape, Alt, Shift, etc that are modifiers are only detected by keydown.  This is a good idea, I've logged it internally as DTM-15862 and we'll add it to the backlog.

**Comment by thebenrob** [comment]
@adobe export issue to Jira project PDCL as Story

**Comment by brenthosie** [comment]
https://github.com/adobe/reactor-extension-core/pull/80
